vrecord

```diff
@@ -1931,6 +1940,7 @@ fi
 while [[ "${RUNTYPE}" = "edit" || "${RUNTYPE}" = "GUI" ]] ; do
     _edit_mode
 done
+WAVEFORM_SCALE_OPTIONS=("digital" "millivolts" "ire")
```

Comment:
needs to be moved to line 1926 to work I think

Reply:
updated